sync/metrics.go

```diff
@@ -53,9 +53,9 @@ func newMetrics() (*metrics, error) {
 		return nil, err
 	}
 
-	unrecentHeader, err := meter.Int64Counter(
-		"hdr_sync_unrecent_header_counter",
-		metric.WithDescription("tracks every time Syncer returns an unrecent header"),
+	outdatedHeader, err := meter.Int64Counter(
+		"hdr_sync_outdated_head_counter",
```

Comment:
@vgonkivs, do we have any panels already? Because this is breaking

Reply:
yes, I will fix them once this PR will be merged